tests/framework/extensions/provisioning/creates.go

```diff
@@ -748,3 +754,174 @@ func createWindowsRegistrationCommand(command, publicIP, privateIP string, machi
 	}
 	return command
 }
+
+// AddRKE2K3SCustomClusterNodes is a helper method that will add nodes to the custom RKE2/K3S custom cluster.
```

Comment:
nit; is this a helper, it seems more like its own function.

Reply:
I can edit it to be more descriptive.